lightwood/analysis/nc/calibrate.py

```diff
@@ -363,19 +371,34 @@ def explain(self, row_insights: pd.DataFrame, global_insights: Dict[str, object]
             if ns.tss.is_timeseries and ns.tss.nr_predictions > 1 and is_numerical:
                 row_insights = add_tn_conf_bounds(row_insights, ns.tss)
 
+            # clip bounds if necessary
+            if is_numerical:
+                lower_limit = 0.0 if ns.positive_domain else -pow(2, 62)
```

Comment:
0 makes sense if positive but -2**64 is pretty intense

Reply:
Yep, though as per above, it's just to avoid overflows.